### PR TITLE
Add Test Experiment to Structured Experiments

### PR DESCRIPTION
**Pull request by nlittlepoole**
# Add Test Experiment to Structured Experiments

Added a test experiment to the `structured_experiments` folder that learners can use as a template for their own experiments. The experiment provides a `Dockerfile` and `docker-compose.yml` which allows it to be run without installing any new dependencies. The experiment does the following:

- Use Tesseract training image as base image
- Convert kaggle label CSV to `gt.txt` text file labels that tesseract expects
- Train tesseract using `kaggle-ground-truth` instead of `scribble-ground-truth`

☝️ The entrypoint for the above is experiment.sh and it calls out to a python script also provided within the experiment folder. That python file handles the conversion of the kaggle labels. 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [x] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [x] Yes
- [ ] No
- [ ] Not necessary

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

**Review by james-caldwell1981 — COMMENTED** [review]
Line 11 in docker-compose.yml is the following:
- "../../data:/train/tesstrain/data"

I'm assuming the colon is a mapping of the local data folder to the train/tesstrain/data folder. What does the hyphen do?

**Comment by nlittlepoole** [comment]
> Line 11 in docker-compose.yml is the following:
> 
> * "../../data:/train/tesstrain/data"
> 
> I'm assuming the colon is a mapping of the local data folder to the train/tesstrain/data folder. What does the hyphen do?

@james-caldwell1981 docker-compose files are actually a [YAML file](https://yaml.org/). In YAML, [the `-` symbol denotes ](https://www.cloudbees.com/blog/yaml-tutorial-everything-you-need-get-started)that it is an item in a list. The `volumes` key can be given multiple volume mounts, not just one. Each would be on a new line starting with `-`. 

Fun fact, all JSON is valid YAML.

**Review by ashtilawat — APPROVED** [review]

**Comment by jacob-torres** [comment]
Is this a foundation to move all training data to Git LFS, rather than S3?

**Comment by jraval** [comment]
We saw this test experiment performed in front of us.  We are using this template for future experiments.  Thanks for your help Niger!

**Review by jraval — COMMENTED** [review]
We saw this test performed in front of us.  It works and the template will  help us significantly.  Thanks for your help Niger.

**Review by jraval — APPROVED** [review]
We saw this test live, there were no issues.  This is something we really needed.  We needed a template for experiments moving forward.  Thank you so much for your help on this.
